doc/dev/delayed-delete.rst

```diff
@@ -2,11 +2,12 @@
  CephFS delayed deletion
 =========================
 
-When you delete a file, the data is not immediately removed. Each
-object in the file needs to be removed independently, and sending
-``size_of_file / stripe_size * replication_count`` messages would slow
-the client down too much, and use a too much of the clients
-bandwidth. Additionally, snapshots may mean some objects should not be
-deleted.
+The deletion of a file does not immediately remove its data. Each
+object in the file must be removed independently. Sending
+``size_of_file / stripe_size * replication_count`` messages slows
+the client unacceptably, and consumes too much client bandwidth. 
+If snapshots exist, their existence can prevent the deletion of 
+objects associated with them.
```

Comment:
If I understand the intent of these sentences, maybe something like

Each of the file's underlying objects must also be explicitly removed.  Doing so immediately would require the client to send ``size_of_file / stripe_size * replication_count`` messages to the Ceph cluster, which would consume significant bandwidth and slow the client unacceptably.